cmd/service/main.go

```diff
@@ -83,6 +86,16 @@ func Run(ctx context.Context, component component.IComponent) error {
 		done <- true
 	}()
 
+	url := fmt.Sprintf(":%v", global.DebugServerPort)
```

Comment:
A HTTP server is already initialised in propeller for prometheus metrics and webscokets. Possible to re-use the same server? It's initialised here: https://github.com/CRED-CLUB/propeller/blob/main/internal/component/api_server_component.go

Reply:
done